packages/node_modules/@webex/plugin-meetings/src/metrics/constants.js

```diff
@@ -44,7 +44,15 @@ const BEHAVIORAL_METRICS = {
   ENABLE_BNR_SUCCESS: 'js_sdk_enable_bnr_success',
   ENABLE_BNR_FAILURE: 'js_sdk_enable_bnr_failure',
   DISABLE_BNR_SUCCESS: 'js_sdk_disable_bnr_success',
-  DISABLE_BNR_FAILURE: 'js_sdk_disable_bnr_failure'
+  DISABLE_BNR_FAILURE: 'js_sdk_disable_bnr_failure',
+  FETCH_MEETING_INFO_V1_SUCCESS: 'js_sdk_fetch_meeting_info_v1_success',
+  FETCH_MEETING_INFO_V1_FAILURE: 'js_sdk_fetch_meeting_info_v1_failure',
+  ADOC_MEETING_INIALIZATION_SUCCESS: 'js_sdk_adoc_meeting_initialization_success',
```

Comment:
As per Arun's suggestion, Need to change ''js_sdk_adoc_meeting_initialization_success'' to 'js_sdk_adoc_meeting_success'

Reply:
Updated.